kapacitor/client.go

```diff
@@ -259,6 +268,11 @@ func (c *Client) Update(ctx context.Context, href string, rule chronograf.AlertR
 		return nil, err
 	}
 
+	prevStatus, err := c.status(ctx, href)
```

Comment:
Makes sense.  Looks like this was an issue in `Update` before this PR.  Perhaps open an issue for this specifically?